charms/slurmctld/src/interface_influxdb.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
```

Comment:
[nit]: Could you add an Omnivector copywrite notice here just so anyone coming in will know that you contributed this interface?